projects/packages/videopress/src/client/lib/url/index.ts

```diff
@@ -129,3 +129,44 @@ export function buildVideoPressURL(
 export const removeFileNameExtension = ( name: string ) => {
 	return name.replace( /\.[^/.]+$/, '' );
 };
+
```

Comment:
Ideally, since we're borrowing this from wp-calypso, could we consider extracting that functionality to a package in the Calypso monorepo and reusing it here, instead of just copying it?